builtin/mainmenu/tab_online.lua

```diff
@@ -174,6 +174,13 @@ local function get_formspec(tabview, name, tabdata)
 				"server_view_clients.png") .. ";btn_view_clients;]"
 		end
 
+		if selected_server.url then
+			retval = retval .. "tooltip[btn_server_url;" .. fgettext("Server Website") .. "]"
+			retval = retval .. "style[btn_server_url;padding=6]"
+			retval = retval .. "image_button[" .. (can_view_clients_list and "4" or "4.5") .. ",1.3;0.5,0.5;" ..
+			core.formspec_escape(defaulttexturedir .. "server_url.png") .. ";btn_server_url;]"
```

Comment:
Misleading indentation. I believe our style guidelines would want this to be indented two times to make clear that it is a continuation of the expression on the previous line. (Though I also see some continuations use a single tab; I don't mind either way.)